samples/account_summaries_list.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+
+# Copyright 2021 Google Inc. All Rights Reserved.
```

Comment:
The copyright holder should be `Google LLC`

Reply:
And all the other files too.

Reply:
Done!